coretex/networking/network_manager_base.py

```diff
@@ -407,7 +410,9 @@ def formData(
             headers = self._headers("multipart/form-data")
             del headers["Content-Type"]
 
-            return self.request(endpoint, RequestType.post, headers, body = params, files = filesData)
+            timeout = 3600 if endpoint in ["upload/chunk", "artifact/upload-file"] else REQUEST_TIMEOUT
```

Comment:
Do not check for endpoints directly since if another one which is uploading a file is used we need to manually enter it here. Use `files` parameter to decide if timeout should be increased or not. Also only read timeout should be increased in upload case right? Not connection timeout.